migration/20170605-move-configuration-links-into-db.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+"""Move links from the Configuration file into the database as ConfigurationSettings
+for the default Library.
+"""
+import os
+import sys
+
+bin_dir = os.path.split(__file__)[0]
+package_dir = os.path.join(bin_dir, "..")
+sys.path.append(os.path.abspath(package_dir))
+
+from core.model import (
+    ConfigurationSetting,
+    Library,
+    get_one_or_create,
+    production_session,
+)
+from api.config import Configuration
+
+Configuration.load()
+_db = production_session()
+library = Library.instance(_db)
+
+for rel, value in (
+        ("terms-of-service", Configuration.terms_of_service_url()),
+        ("privacy-policy", Configuration.privacy_policy_url()),
+        ("copyright", Configuration.acknowledgements_url()),
+        ("about", Configuration.about_url()),
+        ("license", Configuration.license_url()),
+):
```

Comment:
If you use the code these methods use, you can remove the methods themselves in this branch; otherwise we need to wait until everyone is migrated.